samples/dfu/nrf5340/main.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022 INTELLINIUM <giuliano.franchetto@intellinium.com>
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/zephyr.h>
+#include <zephyr/device.h>
+#include <lte_uart_dfu.h>
+#include <zephyr/logging/log.h>
+#include <dfu/dfu_target_mcuboot.h>
+#include <sample_interface.h>
+LOG_MODULE_REGISTER(app, CONFIG_APP_LOG_LEVEL);
+
+K_SEM_DEFINE(exit_sem, 0, 1);
+
+static uint8_t buf[2100];
+
+#if defined(CONFIG_UART_DEVICE_0)
+#define UART_DT		DT_NODELABEL(uart0)
+#elif defined(CONFIG_UART_DEVICE_1)
+#define UART_DT                DT_NODELABEL(uart1)
+#elif defined(CONFIG_UART_DEVICE_2)
+#define UART_DT		DT_NODELABEL(uart2)
+#elif defined(CONCIF_UART_DEVICE_3)
+#define UART_DT		DT_NODELABEL(uart3)
+#else
+#error "No uart device chosen"
+#endif
```

Comment:
This should use a chosen node, not select the UART by doing this

Reply:
I need to check how to do that

Reply:
done